client/blocks/data-center-picker/index.tsx

```diff
@@ -23,297 +16,148 @@ interface ExternalProps {
 		get label(): string {
 			return translate( 'US Central' );
 		},
-		thumbnail: {
-			imageUrl: dfwImg,
-		},
 	},
 	{
 		value: 'dca',
-		name: 'geo_affinity',
 		get label(): string {
 			return translate( 'US East' );
 		},
-		thumbnail: {
-			imageUrl: dcaImg,
-		},
 	},
 	{
 		value: 'ams',
-		name: 'geo_affinity',
 		get label(): string {
 			return translate( 'EU West' );
 		},
-		thumbnail: {
-			imageUrl: amsImg,
-		},
 	},
 ];
 
 const Form = styled.div( {
 	maxWidth: '564px',
-} );
-
-const FormHeadingContainer = styled.div( {
-	display: 'flex',
-	alignItems: 'center',
-	marginBottom: '8px',
-} );
-
-const FormHeading = styled.h3( {
-	fontWeight: 600,
-	marginRight: '8px',
-} );
-
-const FormDescription = styled.p( {
-	marginBottom: '12px',
-} );
-
-const AutomaticFormLabel = styled.label( {
-	display: 'block',
-	marginBottom: '12px',
-	width: '100%',
-	maxWidth: '541px',
-	cursor: 'pointer',
-} );
-
-const CompactLabel = styled.label( {
-	cursor: 'pointer',
-	display: 'flex',
-	padding: '24px 16px',
-	border: '2px solid #DCDCDE',
+	border: '1px solid var( --color-border-subtle )',
 	borderRadius: '4px',
-
-	'&:hover': {
-		background: '#F9FBFD',
-	},
-
-	'&:has(input[type=radio]:checked)': {
-		background: '#F9FBFD',
-		borderColor: '#0675C4',
-	},
+	padding: '24px',
 } );
 
-const PickerOptions = styled.div( { display: 'flex', flexDirection: 'column', gap: '16px' } );
-
-const PickerOption = styled.div( {
-	display: 'flex',
-	flexDirection: 'column',
-	justifyContent: 'flex-start',
-	alignItems: 'flex-start',
-	gap: '8px',
+const FormHidden = styled.div( {
+	fontSize: '0.75rem',
 } );
 
-const PickerOptionLabel = styled.span( { lineHeight: '20px', fontWeight: 600, color: '#101517' } );
-
-const PickerRadio = styled.input( {
-	margin: '2px 16px 0 0',
-} );
+const CustomizeLink = styled.a`
+	text-decoration: underline;
+	cursor: pointer;
+`;
 
-const CompactPickerSelect = styled( FormSelect )( {
-	width: 'fit-content',
-	borderRadius: '4px',
-} );
+const SupportLink = styled.a`
+	text-decoration: underline;
+	color: var( --color-text-subtle ) !important;
+`;
 
-const FormLabelThumbnail = styled.div( {
-	display: 'flex',
-	width: '100%',
-	height: '90px',
-	margin: '2px 0 8px',
-	backgroundColor: 'var(--color-surface)',
-	backgroundPosition: '50% 0',
-	backgroundSize: 'cover',
-	borderRadius: '2px',
-	boxShadow: '0 0 0 1px rgba(var(--color-neutral-10-rgb), 0.5), 0 1px 2px var(--color-neutral-0)',
-	transition: 'all 100ms ease-in-out',
-	overflow: 'hidden',
-	'&:hover': {
-		boxShadow: '0 0 0 1px var(--color-neutral-light), 0 2px 4px var(--color-neutral-10)',
-	},
-} );
-
-const FormLabelThumbnailImg = styled.img( {
-	objectFit: 'cover',
-} );
+const StyledSelectControl = styled( SelectControl )`
+	.components-input-control__backdrop {
+		border-color: var( --studio-gray-10 ) !important;
+	}
+`;
 
-const RECOMMENDED_DATA_CENTER = '';
+const StyledLabel = styled.div`
+	text-transform: none;
+	font-size: 0.875rem;
```

Comment:
Do we have vars for things like font sizes too?

Reply:
We do, but they're not working here, I'm not sure why. They may not be loaded here:

https://github.com/Automattic/wp-calypso/blob/7c0a42dda287bd6b8474d9d29eedb346a641e258/packages/typography/styles/variables.scss#L38-L48